contents/docs/integrate/_snippets/install-python.mdx

```diff
@@ -0,0 +1,33 @@
+<PythonInstall />
+
+```bash
```

Comment:
Starts abruptly compared to others, but not sure if because of the `<PythonInstall />` component above.